examples/README.md

```diff
@@ -28,7 +28,7 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
   * [hpc-slurm-sharedvpc.yaml](#hpc-slurm-sharedvpcyaml-) ![community-badge]
   * [client-google-cloud-storage.yaml](#client-google-cloud-storageyaml--) ![community-badge] ![experimental-badge]
   * [hpc-slurm-gromacs.yaml](#hpc-slurm-gromacsyaml--) ![community-badge] ![experimental-badge]
-  * [omnia-cluster.yaml](#omnia-clusteryaml--) ![community-badge] ![experimental-badge]
+  * [omnia-cluster.yaml](#omnia-clusteryaml--) ![community-badge] ![experimental-badge] ![deprecated-badge]
```

Comment:
I think the anchor link will now be broken. I think it wants one `-` for every badge.

Reply:
That explains the mystery of the dashes. I had noticed it but blamed it on poor rendering in GitHub PR UI. Fixed.